### PR TITLE
Autogen code bilinear_tensor_product

### PR DESCRIPTION
**Pull request by gouzil**
### PR types

Others
### PR changes

Others
### Describe

autogen code bilinear tensor product

### CODE REVIEW

File: paddle/phi/api/yaml/ops.yaml [hunk 1]
```diff
@@ -206,6 +206,16 @@
   data_transform :
     skip_transform : out_size, size_tensor, scale_tensor
 
+- op : bilinear_tensor_product
+  args : (Tensor x, Tensor y, Tensor weight, Tensor bias)
+  output : Tensor
+  infer_meta :
+    func : BilinearInferMeta
```
Comment (heavyrain-lzy) [inline paddle/phi/api/yaml/ops.yaml:213]: 请确认一下这里的infermeta是`BilinearInferMeta`还是`BilinearTensorProductInferMeta`

Comment (gouzil) [reply paddle/phi/api/yaml/ops.yaml:213]: 从`paddle/fluid/operators/bilinear_tensor_product_op.cc`来看是`BilinearInferMeta`
```c++
DECLARE_INFER_SHAPE_FUNCTOR(bilinear_tensor_product,
                            BilinearTensorProductInferShapeFunctor,
                            PD_INFER_META(phi::BilinearInferMeta));

```

Comment (gouzil) [reply paddle/phi/api/yaml/ops.yaml:213]: https://github.com/PaddlePaddle/Paddle/blob/61fe2198185bb92e43eb846617b5faf5d0e40eb9/paddle/phi/infermeta/multiary.h#L201

**Review by heavyrain-lzy — APPROVED** [review]
LGTM

**Pull request merged**